docs/config-file-ort-yml.md

```diff
@@ -4,7 +4,8 @@ The items below can be configured by adding an `.ort.yml` file to the root of th
 
 * [excludes](#excludes) - Mark [files, directories](#excluding-paths) or [package manager scopes](#excluding-scopes) as
   not included in released artifacts.
-* [license finding curations](#curations) - Overwrite scan results to correct identified licenses.
+* [curations](#curations) - Overwrite scan results to correct identified licenses or correct metadata of third-party
```

Comment:
How about stealing the first sentence from `config-file-curations-yml.md´? e.g. "Curations correct invalid or missing package metadata and set the concluded license for packages."